runtime/doc/autocmd.txt

```diff
@@ -831,16 +831,16 @@ ShellFilterPost			After executing a shell command with
 				":{range}!cmd", ":w !cmd" or ":r !cmd".
 				Can be used to check for any changed files.
 							*SourcePre*
-SourcePre			Before sourcing a Vim script. |:source|
+SourcePre			Before sourcing a Vim script or lua. |:source|
```

Comment:
are we using "a vim script" otherwhere. otherwise "Before sourcing a vim or lua file" maybe?

Reply:
how about "Before sourcing a vim script or lua file" ?

Reply:
better, but are we (or even recently written vim help) using "a vim script" to refer to a .vim file?

Reply:
I don't know if we are using it. It was 'a vim script' before that's why I said .

Reply:
yes it was _here_ before but neither `:runtime` nor `:source` uses it, they just refer to vim/lua files.

Reply:
Ok then 'vim/lua' files it is

Reply:
Changed to "Before sourcing a vim/lua file"